src/modules/UI/selectors.js

```diff
@@ -112,3 +112,29 @@ export const getSceneState = (state: State, sceneKey: string) => {
   const sceneState = getScenesState(state)[sceneKey]
   return sceneState
 }
+
+export const getExchangeRate = (state: State, fromCurrencyCode: string, toCurrencyCode: string): number => {
+  const fromCurrencyCleaned = fromCurrencyCode.replace('iso:', '')
```

Comment:
Selector for getting exchange rates from the redux store

Reply:
Move `calculateSettingsFiatFromCrypto` to wallet selectors. Make `state` the first parameter. Rename function name and consider combining and making the third parameter the fiat currency code that you want to use.